draft-irtf-cfrg-frost.md

```diff
@@ -1093,6 +1093,7 @@ The procedure for committing to a polynomial `f` of degree `t-1` is as follows.
 ~~~
 
 The procedure for verification of a participant's share is as follows.
+If `vss_verify` fails, the participant MUST abort the protocol, and failure should be investigated out of band.
```

Comment:
This seems like text that already exists above:

> If the validation fails, the participant can issue a complaint
> against the dealer, and take actions such as broadcasting this complaint to all
> other participants. We do not specify the complaint procedure in this draft, as
> it will be implementation-specific.

Can we update the text there instead?

Reply:
Yeah agreed

Reply:
Removed the other text as broadcast rounds/complaints/etc is 1) hard to get right, and 2) very out of scope for this draft. We now only specify that if this check fails, the participant MUST abort and failure should be investigated out of band.